src/store/store.js

```diff
@@ -114,7 +115,23 @@ export function stopSync() {
   // TODO
 }
 
-export function start() {
+function newWalletVersionCheck() {
+  getWalletVersion().then((versionDetails) => {
+    if (!versionDetails.isLatest) {
+      const params = [
+        `A new version of Emerald Wallet is available (${versionDetails.tag}).`,
+        'info',
+        20 * 1000,
```

Comment:
20 seconds is good i guess?

Reply:
yeah. IMO it should just be forever, as its dismissed on click away anyhow.

Reply:
but 20s should be fine  right?